src/components/Accordion/accordionChangelogData.ts

```diff
@@ -14,7 +14,10 @@ export const changelogData: ChangelogData[] = [
     version: "3.1.4",
```

Comment:
Should this version number be updated?

Reply:
Ah, good catch.  Those should be split into two separate entries.  I made the adjustment and updated the PR.